x-pack/test/ingest_manager_api_integration/apis/file.ts

```diff
@@ -47,7 +47,7 @@ export default function ({ getService }: FtrProviderContext) {
             '/api/ingest_manager/epm/packages/filetest/0.1.0/kibana/visualization/sample_visualization.json'
           )
           .set('kbn-xsrf', 'xxx')
-          .expect('Content-Type', 'application/json; charset=utf-8')
+          .expect('Content-Type', 'text/plain; charset=utf-8')
```

Comment:
Why are these tests changed? Did we explicitly change these types elsewhere?

Reply:
Yes, the package registry changed its behaviour, this is tracked in https://github.com/elastic/package-registry/issues/590 

We just pass these types through, our own code doesn't depend on these mime types. So, there is no real reason to keep these tests right now, but they serve as implementation examples how to run tests against custom packages in `apis/fixtures`, so I'd prefer to keep them.

Reply:
Thanks for the link, I can add my comments there.

Agreed on keeping them, or I wouldn't have noticed the change in registry behavior.